source/views/index.html.haml

```diff
@@ -42,57 +42,76 @@ title: Refresh the Triangle
+        product. We will look at how product teams can make usage data
+        actionable, and ways that they can gather data and calculate metrics
+        whether using our platform or not.
 
-      %h3 The Speakers:
+      %p
+        %strong Michael Peach
+        is the Head of Product Marketing for
+        = link_to "Pendo", "http://pendo.io"
+        where he leads messaging, positioning, and launch activities for Pendo’s
+        product experience platform.  Prior to joining Pendo, he was a program
+        director at IBM where he led marketing and demand generation teams for
+        their mobile, application integration and business process management
+        portfolios.  He has also held product management and business
+        development roles for several small and early-stage technology
+        companies.  He has a bachelor’s degree from Santa Clara University and
+        an MBA from the University of North Carolina.
+
+      %h4 Help Your Client Identify KPIs Based on their Business Goals
 
-      %h4 Dana Calder
       %p
-        Dana fell in love with North Carolina while studying Sociology at Salem
-        College. Prior to joining The Iron Yard, she spent more than eight years
-        in educational sales and support. Dana loves exploring the Durham food
-        scene with her wife and son.
+        Every business has goals. Businesses that make investments in products
+        and services expect some sort of return. For some, the problem is
+        knowing what to measure to determine if that investment is producing a
+        return.
 
-      %h4 Ellen Gowdy
       %p
-        Ellen's sweet spot lies in helping companies grow from 20 to 100+ people
-        while preserving and enhancing culture. She's a change almanac, and is
-        skilled at cultivating systems that help companies thrive as they grow.
-        As the Growth Manager at CRISP agency in Raleigh, Ellen puts the right
-        people in the right seats, scales CRISP's unique company culture and
-        helps build an iconic brand known throughout the region. Outside the
-        office, she enjoys nature, cooking, documentaries and personal finance
-        (no, really!)
+        %strong Greg Hyer,
+        Marketing Communications Manager at
+        = link_to "TheeDesign", "https://www.theedesign.com/"
+        and Co-host of the Social
+        Selling Podcast by
+        = succeed ',' do
+          = link_to "Linking into Sales", "http://www.linkingintosales.com/"
+        will be sharing several ways you
+        can help your clients identify KPIs they can measure based on their
+        business goals and objectives. He will focus on three specific business
+        models; retail, SaaS and business services, each of which Greg has
+        direct working experience in.
 
     %p
-      Parking can be found at CRISP, and there is plenty of it.
+      Parking can be found on the street and in the parking garages by the
```

Comment:
Let's say "Parking can be found on the street or in the Wake County Justice Center Deck at 216 W. Cabarrus Street."